R/utils-limits.R

```diff
@@ -47,14 +46,12 @@ testCallouts <- function(plot_obj, xlimits){
 #' 
 testCalloutsByView <- function(plot_obj, view_num, xlimits_real, width_char, xrange){
   
-  i_view <- which(names(plot_obj$view_num) == "callouts")
+  i_view <- which(names(plot_obj[[view_num]]) == "callouts")
   
   for(i in i_view){
     callout_args <- plot_obj[[view_num]][[i]]
-    if (!is.na(xtfrm(callout_args$x[i])) |  
-        !is.na(xtfrm(callout_args$y[i])) |
-        is.null(xtfrm(callout_args$x[i])) |  
-        is.null(xtfrm(callout_args$y[i]))) {  
+    
+    if (!isEmptyOrBlank(callout_args$x)) {  
```

Comment:
Should this still be checking `callout_args$x[i]` since it's within the for loop?